src/transformState.js

```diff
@@ -46,7 +46,34 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  const OPERATIONS = {
+    add: 'addProperties',
+    remove: 'removeProperties',
+    clear: 'clear',
+  };
+
+  for (let i = 0; i < transforms.length; i++) {
+    switch (true) {
+      case (transforms[i].operation === OPERATIONS.add):
+        for (const keys in transforms[i].properties) {
```

Comment:
якщо перебиратимеш через for of
```suggestion
        for (const keys in transform.properties) {
```